redirects-transition.conf

```diff
@@ -1123,6 +1123,9 @@ rewrite ^/fecig/(.*) https://www.fec.gov/office-inspector-general/ redirect;
 # fecletter/ broader redirects
 rewrite ^/fecletter/(.*) https://www.fec.gov/data/filings/?data_type=processed&form_type=RFAI redirect;
 
+# foia/ broader redirects
+rewrite ^/foia/quarterly/FEC-([0-9]{4})-Q([0-4]{1}).zip https://www.fec.gov/resources/cms-content/documents/FEC-$1-Q$2.zip redirect;
```

Comment:
@dorothyyeager I just tested this on my computer and it doesn't seem to like the {4} and {1}. I'm not sure why. Can you replace those with a `+` instead? That worked when I tested locally. So it would look like:

```
rewrite ^/foia/quarterly/FEC-([0-9]+)-Q([0-4]+).zip https://www.fec.gov/resources/cms-content/documents/FEC-$1-Q$2.zip redirect;
```

Reply:
Done, thanks @patphongs